frontend/src/app/components/MainInstallButton.js

```diff
@@ -54,6 +54,7 @@ export default class MainInstallButton extends React.Component {
       installButton = this.renderOneClickInstallButton(experimentTitle);
     } else {
       installButton = <span className="default-btn-msg" data-l10n-id="landingInstallButton">
+        Install the Test Pilot Add-on
```

Comment:
Yes, it is, but now that we no longer have react-router trying to change the nodes that l20n touched, it isn't a problem any more.

Unless the page itself tries to change some nodes after l20n touches them, which isn't the case here.